Monika After Story/game/script-topics.rpy

```diff
@@ -8511,6 +8511,244 @@ label monika_citizenship:
     m 1eka "Thank you for listening to me vent, [player]."
     return
 
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_idle_game",category=['games'],prompt="I'm going to game for a bit",random=False,pool=True))
+
+label monika_idle_game:
+    m 1eub "That sounds fun!"
+    menu:
+        m "What kind of game are you going to play?"
+        "A competetive game.":
+            m 1eua "That sounds like it could be fun!"
+            m 3lksdla "I can be pretty competitive myself."
+            m 3eua "So I know just how stimulating it can be to face a worthy opponent."
+            m 2hksdlb "And sometimes frustrating when things don't go right."
+            m 2hua "Anyway, I'll let you get on with your game."
+            m 2hub "I'll try not to bother you until you finish, but I can't blame you if you get distracted by your lovely girlfriend~ Ahaha~"
+            m 1hub "I'm rooting for you, [player]!"
+            # set return label when done with idle
+            $ mas_idle_mailbox.send_idle_cb("monika_idle_game_competetive_callback")
+        "A game just for fun.":
+            m 1eud "A game just for having fun?"
+            m 1lksdla "Aren't most games made to be fun?"
+            m 3eub "Anyway, I'm sure you could do all sorts of fun things in a game like that."
+            m 1ekbla "I really wish I could join you and we could have fun together."
+            m 1lksdla "But for now, I'll leave you to it."
+            m 1hub "Have fun, [player]!"
+            # set return label when done with idle
+            $ mas_idle_mailbox.send_idle_cb("monika_idle_game_fun_callback")
+        "A story driven game.":
+            m 1sub "Oh?"
+            m "That sounds really interesting!"
+            m 1ekbsa "Gosh, I really wish I could be there with you to experience it together."
+            m 1hksdlb "Maybe I {i}can{/i} experience it with you if I really tried."
+            show monika 5eua at t11 zorder MAS_MONIKA_Z with dissolve
+            m 5eua "I guess you could call it looking over your shoulder. Ehehe~"
+            m "You can go ahead and start it now. I'll try not to break anything by trying to watch."
+            # set return label when done with idle
+            $ mas_idle_mailbox.send_idle_cb("monika_idle_game_story_callback")
+        "A skill/practice based game.":
```

Comment:
Probably should keep menu options as a way we would say them.
```suggestion
        "A skill or practice based game.":
```

Reply:
I thought having a "/" would be more natural. Personally, I'm more likely to say "/" than "or". 

The way I see it, saying "or" can kind of mean you're referring to more than one thing, like you're either going to do this or you're going to do something completely different, and saying "/" would mean that the terms are pretty much interchangeable and mean the same thing in that context. 

I guess saying "or" kind of makes sense, but it doesn't sound natural to me. Maybe I could replace it with the actual word "slash"?